docs/source/usage/param.rst

```diff
@@ -3,6 +3,36 @@
 .param Files
 ============
 
+.. sectionauthor:: Axel Huebl
+
+Parameter files, ``*.param`` placed in ``include/simulation_defines/param/`` are used to set all **compile-time options** for a PIConGPU simulation.
+This includes most fundamental options such as numerical solvers, floating precision, memory usage due to attributes and super-cell based algorithms, density profiles, initial conditions etc.
+
+Rationale
+---------
+
+High-performance hardware comes with a lot of restrictions on how to use it, mainly memory, control flow and register limits.
+In order to create an efficient simulation, PIConGPU compiles to **exactly** the numerical solvers (kernels), physical attributes (fields, species) for the setup you need to run which will furthermore be specialized for a specific hardware.
```

Comment:
REQUIRED:
I think a `and` is missing here:
PIConGPU compiles exactly the numerical solvers (kernels) **and** physical attributes (fields, species) for the setup you need
otherwise it is a list without "and"